pkg/controller/operators/catalog/subscription/syncer.go

```diff
@@ -45,34 +45,37 @@ func (s *subscriptionSyncer) now() *metav1.Time {
 // Sync reconciles Subscription events by invoking a sequence of reconcilers, passing the result of each
 // successful reconciliation as an argument to its successor.
 func (s *subscriptionSyncer) Sync(ctx context.Context, event kubestate.ResourceEvent) error {
-	res := &v1alpha1.Subscription{}
-	if err := scheme.Convert(event.Resource(), res, nil); err != nil {
+	sub := &v1alpha1.Subscription{}
```

Comment:
these changes are unrelated to the bug (and pretty subjective), can you remove them so that we can focus on the fix when we are backporting?

Reply:
@ecordell removed the changes (although I'd have liked to discuss the pros and cons of making changes like these if we had a bit more time)